### PR TITLE
Do not rewrite SELinux labels on volume mounts

### PR DESCRIPTION
**Pull request by superseb**
Requires https://github.com/rancher/rancher-selinux/pull/6

### CODE REVIEW

File: cluster/plan.go [hunk 1]
```diff
@@ -479,11 +479,11 @@ func (c *Cluster) BuildKubeletProcess(host *hosts.Host, serviceOptions v3.Kubern
 		}
 	} else {
 		Binds = []string{
-			fmt.Sprintf("%s:/etc/kubernetes:z", path.Join(host.PrefixPath, "/etc/kubernetes")),
+			fmt.Sprintf("%s:/etc/kubernetes", path.Join(host.PrefixPath, "/etc/kubernetes")),
 			"/etc/cni:/etc/cni:rw,z",
 			"/opt/cni:/opt/cni:rw,z",
```
Comment (cmurphy) [inline cluster/plan.go:484]: There are still a few volumes like these that are using the `z` mount option, can these be fixed too?

**Review by cmurphy — COMMENTED** [review]
I don't know if the goal is to remove some of them or all of them but there's a few more sprinkled around:

https://github.com/rancher/rke/blob/462a7ba01ebb38d2f45e4a7b96e1c86c7a774b4f/hosts/hosts.go#L309
https://github.com/rancher/rke/blob/462a7ba01ebb38d2f45e4a7b96e1c86c7a774b4f/cluster/file-deployer.go#L63
https://github.com/rancher/rke/blob/462a7ba01ebb38d2f45e4a7b96e1c86c7a774b4f/pki/deploy.go#L89
https://github.com/rancher/rke/blob/462a7ba01ebb38d2f45e4a7b96e1c86c7a774b4f/pki/pki.go#L115
https://github.com/rancher/rke/blob/462a7ba01ebb38d2f45e4a7b96e1c86c7a774b4f/services/controlplane.go#L383
https://github.com/rancher/rke/blob/462a7ba01ebb38d2f45e4a7b96e1c86c7a774b4f/services/etcd.go#L350

**Review by cmurphy — COMMENTED** [review]
lgtm :+1:

**Review by kinarashah — APPROVED** [review]
Changes LGTM, @superseb all clusters will start upgrading because of binds change, right? we'd need to call this out.

**Pull request merged**